src/source/src/table_v2.rs

```diff
@@ -17,7 +17,7 @@ use std::sync::RwLock;
 use rand::prelude::SliceRandom;
 use risingwave_common::array::StreamChunk;
 use risingwave_common::catalog::{ColumnDesc, ColumnId};
-use risingwave_common::error::Result;
+use risingwave_common::error::Result as RwResult;
```

Comment:
what the pr intends to do is eliminate usage of `RwError`. You may need a new result type.